src/features/subscription/NotificationsSettingsModal.tsx

```diff
@@ -0,0 +1,136 @@
+import { Close } from 'ui/svg/icons/Close'
+import { EmailFilled } from 'ui/svg/icons/EmailFilled'
+import { Invalidate } from 'ui/svg/icons/Invalidate'
+import { Spacer, Typo } from 'ui/theme'
+
+interface Props {
+  visible: boolean
+  dismissModal: () => void
+  theme: SubscriptionTheme
+  onPressSaveChanges: ({
+    allowEmails,
+    allowPush,
+  }: {
+    allowEmails: boolean
+    allowPush: boolean
+  }) => void
+}
+
+export const NotificationsSettingsModal: FunctionComponent<Props> = ({
+  visible,
+  dismissModal,
+  theme,
+  onPressSaveChanges,
+}) => {
+  const [settings, setSettings] = useState({ allowEmails: false, allowPush: false })
+
+  const {
+    visible: isPushModalVisible,
+    showModal: showPushModal,
+    hideModal: hidePushModal,
+  } = useModal(false)
+
+  const { pushPermission } = usePushPermission()
+  const isPushPermissionGranted = pushPermission === 'granted'
+
+  const togglePush = () => {
+    if (isPushPermissionGranted) {
+      setSettings((prevState) => ({ ...prevState, allowPush: !prevState.allowPush }))
+    } else {
+      showPushModal()
+    }
+  }
+
+  const onRequestNotificationPermissionFromModal = () => {
+    hidePushModal()
+    Linking.openSettings()
+  }
+
+  return (
+    <AppModal
+      visible={visible}
+      title={`S’abonner au thème “${mapSubscriptionThemeToName[theme]}”`}
+      rightIconAccessibilityLabel="Ne pas s’abonner"
+      rightIcon={Close}
+      onRightIconPress={dismissModal}>
+      <ModalContent>
+        <Typo.Body>Pour recevoir toute l’actu de ce thème, tu dois, au choix&nbsp;:</Typo.Body>
+
+        <Spacer.Column numberOfSpaces={6} />
+
+        <SectionWithSwitch
+          title="Autoriser l’envoi d’e-mails"
+          icon={EmailFilled}
+          active={settings.allowEmails}
+          toggle={() =>
+            setSettings((prevState) => ({ ...prevState, allowEmails: !prevState.allowEmails }))
+          }
+        />
+        {Platform.OS !== 'web' && (
+          <React.Fragment>
+            <Spacer.Column numberOfSpaces={6} />
+
+            <SectionWithSwitch
+              title="Autoriser les notifications"
+              icon={StyledBellFilled}
+              active={settings.allowPush}
+              toggle={togglePush}
+            />
+          </React.Fragment>
+        )}
+        <Spacer.Column numberOfSpaces={6} />
+
+        <StyledCaption>Tu pourras gérer tes alertes depuis ton profil.</StyledCaption>
+
+        <Spacer.Column numberOfSpaces={6} />
+
+        <ButtonPrimary
+          wording="Valider"
+          disabled={!settings.allowEmails && !settings.allowPush}
+          onPress={() => {
+            dismissModal()
+            onPressSaveChanges(settings)
+          }}
+        />
+        <Spacer.Column numberOfSpaces={6} />
+
+        <ButtonTertiaryBlack
+          wording="Tout refuser et ne pas recevoir d’actus"
+          icon={Invalidate}
+          onPress={dismissModal}
+        />
```

Comment:
L'espace entre les boutons me paraît être plus grand que ce qu'il faut, pour y remédier :
```suggestion
        <ButtonTertiaryBlack
          wording="Tout refuser et ne pas recevoir d’actus"
          icon={Invalidate}
          onPress={dismissModal}
          inline
        />
```